pkg/client/cache/delta_fifo.go

```diff
@@ -522,6 +522,23 @@ func (f *DeltaFIFO) syncKey(key string) error {
 		return nil
 	}
 
+	// If we are doing Resync() and there is already an event queued for that object,
```

Comment:
I think there is only one such place (except from DeltaFIFO). But that's the main reason why I don't want to change it for 1.4.